apptest.go

```diff
@@ -190,7 +185,7 @@ func (a *AppSetup) InstallApps(ctx context.Context, apps []App) error {
 func (a *AppSetup) UpgradeApp(ctx context.Context, current, desired App) error {
 	var err error
 
-	err = a.createAppCatalogs(ctx, []App{current, desired})
+	err = a.createCatalogs(ctx, []App{current, desired})
```

Comment:
We replaced the AppCatalog CRD with the Catalog CRD. 

During the migration period we created both AppCatalog and Catalog CRs but the `UpgradeApp` test got missed which was a bug.

The migration is over so we now only need to create Catalog CRs.